lib/logger/lib/logger/backends/console.ex

```diff
@@ -1,5 +1,11 @@
 defmodule Logger.Backends.Console do
-  @moduledoc false
+  @moduledoc """
+  This module is the default event handler for `Logger` used to write
+  messages to the console. It implements the
+  [`:gen_event`](https://erlang.org/doc/man/gen_event.html) behaviour.
+  This module can be used as a reference those who need to implement
+  a custom Logger backend.
+  """
```

Comment:
Let's not make it public given the official API to use it is simply `console`. :) We should probably fix the link instead.

Reply:
Should it link to the raw file in Github?

Reply:
We should not link to it at all. :) Can you please tell us where the link is?

Reply:
I linked to the file...

Reply:
The bad link is on https://hexdocs.pm/logger/Logger.html#module-custom-backends . -- search for "Logger.Backends.Console" and you'll find that ExDoc has auto-created a link to https://hexdocs.pm/logger/Logger.Backends.Console.html but it's a 404.

Coming across a 404 when trying to wrap your head around Logger event handlers is really confusing, as is the fact that this module implements callbacks from ERLANG (!).  I added a link to the Erlang page in the GenEvent module to help alleviate the confusion.

Reply:
Oh, so we can leave your comments as code comments, just not a public documentation. That should help those reading the source. WDYT?

Reply:
I think it will be Ok with the link to the Github file since it's really only developers who would be looking at that.